autoload/db.vim

```diff
@@ -187,6 +196,7 @@ function! s:filter_write(url, in, out, mods, bang, prefer_filter) abort
   if !a:bang
     let t:db_job_running = 1
   endif
+  doautocmd User DBQueryStart
```

Comment:
I think we want `<nomodeline>` for these.

Reply:
Fixed.